client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -72,21 +84,28 @@ export const useInitialiseForm = () => {
       url,
       username,
     };
-
-    await update(syncSettings).catch(e => {
-      console.error(e);
-      setError({ message: 'Unable to save settings' });
-      setIsLoading(false);
+    try {
+      await update(syncSettings);
+      await restart().catch(e => {
+        console.error(e);
+        const message = parseErrorMessage(
+          e as Error,
+          'error.unable_to_restart_server'
+        );
+        setError({ message });
+        setIsLoading(false);
+      });
       return;
```

Comment:
Hmm, this will return from `catch` statement, and continue on with setting polling timeout etc.. should it be two `try/catch` statements one for update and another for restart ?

Reply:
Hmm on second look, this `return` would just exist onSave and would stop polling, @roxy-dao does this branch actually work if correct credentials are present ?

I was thinking this: https://github.com/openmsupply/open-msupply/compare/523-relocate-query-to-get-site-id-before-going-out-of-bootstrap-mode...523-error-handle-example

Reply:
It works but I can swap it to 2 try catch if you want 😄